web-wallet/src/lib/components/Allocate/Allocate.svelte

```diff
@@ -399,10 +389,6 @@
     font-weight: bold;
   }
 
-  :global(.operation__send-address--invalid) {
-    color: var(--error-color);
-  }
```

Comment:
This is not needed, as in other views we have
```
:global(&__input-field:invalid) {
  color: var(--error-color);
}
```

I am thinking we should just move the styling, ie `input:invalid` to the global css?

cc: @kieranhall @ascartabelli

Reply:
Have created a refactoring issue: https://github.com/dusk-network/rusk/issues/2485